fastlane/Fastfile

```diff
@@ -584,7 +592,16 @@ REPOSITORY_NAME="WordPress-Android"
         UI.user_error! "Abort." unless UI.confirm(error_message)
       else
         UI.message("Strings.xml file for #{lib[:name]} downloaded to #{download_path}.")
-        sh(merge_strings_tool_file_path, lib[:name], download_path)
+        if lib.key?(:merge_tool)
+          sh(lib[:merge_tool], lib[:name], download_path) 
```

Comment:
Currently, we have two tools for importing strings in WPAndroid: `an_localize_libs` in the `release-toolkit` and the `merge_strings.xml.py` script. I think that a good next step would be to switch to a single tool for this task. 
I think that `an_localize_libs` should be able to handle the Gutenberg file with a very few changes, so this is probably the best solution.